### PR TITLE
Persist sort settings

### PR DESCRIPTION
**Pull request by broskoski**
This can be overridden by having a query param in the url, but the cookie only changes when you click the sort link. That make sense?

### CODE REVIEW

File: lib/middleware/sort.coffee [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Set sort from cookie
+#
+module.exports = (req, res, next) ->
+  # if the sort cookie is set, use that, otherwise default to updated_at
+  sort = if req.cookies['sort']? then req.cookies['sort'] else 'updated_at'
```
Comment (dzucconi) [inline lib/middleware/sort.coffee]: `sort = req.cookies.sort or 'updated_at'` (unless sort is truthy)

File: models/state.coffee [hunk 2]
```diff
@@ -6,13 +6,15 @@ module.exports = class State extends Backbone.Model
...
   initialize: ->
     # set values from cookies
     each keys(@defaults), (key) =>
       @set(key, val) if val = cookies.get key
 
     @on 'change:view_mode', @setCookie
+    @on 'change:sort', @setCookie
 
   setCookie: (model, value)->
     cookies.set keys(model.changed)[0], value
```
Comment (dzucconi) [inline models/state.coffee]: This is fine, but this model should probably be named something more specific like 'ViewState' or 'UIState'

Comment (broskoski) [inline models/state.coffee]: Can do this

File: lib/middleware/sort.coffee (hunk 1 above)
Comment (broskoski) [inline lib/middleware/sort.coffee]: Tru

**Pull request merged**